doc/source/tutorials.rst

```diff
@@ -0,0 +1,242 @@
+   AWS = BraketClientBackend(device = device, verbatim_circuit=False)
+
+   counts = AWS.execute_circuit(c, nshots=1000).frequencies()
+   print(counts)
+
+Alternatively, one can also use the LocalSimulator to execute circuit `c`. This can be done by leaving `device` empty as it defaults to the LocalSimulator.
+
+.. code-block:: python
+
+   AWS = BraketClientBackend(verbatim_circuit=False)
+
+   counts = AWS.execute_circuit(c, nshots=1000).frequencies()
+   print(counts)
+
+
+To monitor the status of a circuit that is executed, especially on an Amazon Braket device, one can set `monitoring=True`. By default, `monitoring=False`.
+
+.. code-block:: python
+
+   device = "arn:aws:braket:eu-north-1::device/qpu/iqm/Garnet"
+   AWS = BraketClientBackend(device = device, verbatim_circuit=False, monitoring=True)
+
+   counts = AWS.execute_circuit(c, nshots=1000).frequencies()
+   print(counts)
+
+
+Submit a circuit in verbatim mode
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In verbatim mode, the circuit is executed on the device without any transpilation. The user has to ensure that the circuit is specifically written in the device's native gates and gates respect the topology of the device.
+Therefore, before submitting a Qibo circuit in verbatim mode, it is recommended to extract the Amazon Braket device's information. We will demonstrate this below.
+
+Extracting Amazon Braket device parameters
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The Amazon Braket devices can be found here: https://aws.amazon.com/braket/quantum-computers/. Using the `IQM Garnet device <https://aws.amazon.com/braket/quantum-computers/iqm/>`_ as an example, we demonstrate how to extract useful device information such as the qubit topology and native gates. The qubit connectivity on the IQM Garnet device can be visualised using `NetworkX <https://networkx.org/>`_.
+
+.. code-block:: python
+
+   import networkx as nx
+
+   device = "arn:aws:braket:eu-north-1::device/qpu/iqm/Garnet"
+   connectivity_graph = AwsDevice(device).properties.paradigm.connectivity.connectivityGraph
+   native_gates = AwsDevice(device).properties.paradigm.nativeGateSet
+   print(native_gates)
+
+   G = nx.Graph()
+   for node, neighbors in connectivity_graph.items():
+      for neighbor in neighbors:
+         G.add_edge(node, neighbor)
+
+   nx.draw(G, pos=nx.spring_layout(G), with_labels=True, node_color='lightblue', node_size=500, font_size=10, font_weight='bold', edge_color='gray')
+
+
+Submit the circuit
+~~~~~~~~~~~~~~~~~~
+
+Let us run a circuit with `verbatim_circuit=True` on an Amazon Braket device, using IQM Garnet as an example. Here, when `verbatim_circuit=True`, the circuit is submitted as is onto the Amazon Braket device. The device expects to receive a circuit that is written in native gates with qubits in the range of the device. For IQM Garnet, the native gates are `CZ` and `PRX` gates. IQM Garnet has qubits indexed from 1 to 20.
```

Comment:
```suggestion
```
this is repeated almost identical in the following line

Reply:
Oops. Thanks for spotting that!